asab/web/authz/service.py

```diff
@@ -24,29 +34,40 @@ class AuthzService(asab.Service):
 
 	def __init__(self, app, service_name="asab.AuthzService"):
 		super().__init__(app, service_name)
+		self._DisableTokenVerification = asab.Config.getboolean("authz", "_disable_token_verifiction")
+		if jwcrypto is None and not self._DisableTokenVerification:
+			raise ModuleNotFoundError(
+				"You are trying to use asab.web.authz without 'jwcrypto' installed. "
+				"Please run 'pip install jwcrypto' "
+				"or install asab with 'authz' optional dependency.")
+		self.PublicKeysUrl = asab.Config.get("authz", "public_keys_url")
+		if len(self.PublicKeysUrl) == 0 and not self._DisableTokenVerification:
+			raise ValueError("No public_keys_url provided in [authz] config section.")
+		self.PublicKey = None  # TODO: Support multiple public keys
+		self.App.PubSub.subscribe("Application.tick/10!", self.initialize)
```

Comment:
- both initialize() and the pubsub trigger now call the _fetch_keys() method.
- the if-ready logic was moved into _fetch_keys().
- trigger period reduced to 30 seconds (it should be quite frequent since an app that relies on authz cannot fully function without the public key.